bug_1292234.patch

```diff
@@ -306,6 +306,111 @@ diff --git a/devtools/server/actors/storage.js b/devtools/server/actors/storage.
  StorageActors.createActor({
    typeName: "Cache",
  }, {
+diff --git a/devtools/server/tests/browser/browser.ini b/devtools/server/tests/browser/browser.ini
+--- a/devtools/server/tests/browser/browser.ini
++++ b/devtools/server/tests/browser/browser.ini
+@@ -122,6 +122,7 @@ skip-if = e10s # Bug 1183605 - devtools/
+ [browser_storage_listings.js]
+ [browser_storage_updates.js]
+ skip-if = (verify && debug && (os == 'mac' || os == 'linux'))
++[browser_storage_webext_storage_local.js]
+ [browser_stylesheets_getTextEmpty.js]
+ [browser_stylesheets_nested-iframes.js]
+ [browser_register_actor.js]
+diff --git a/devtools/server/tests/browser/browser_storage_webext_storage_local.js b/devtools/server/tests/browser/browser_storage_webext_storage_local.js
+new file mode 100644
+--- /dev/null
++++ b/devtools/server/tests/browser/browser_storage_webext_storage_local.js
+@@ -0,0 +1,89 @@
++/* vim: set ft=javascript ts=2 et sw=2 tw=80: */
++/* Any copyright is dedicated to the Public Domain.
++   http://creativecommons.org/publicdomain/zero/1.0/ */
++
++/* import-globals-from storage-helpers.js */
++
++"use strict";
++
++Services.scriptloader.loadSubScript(
++  "chrome://mochitests/content/browser/devtools/server/tests/browser/storage-helpers.js",
++  this);
++
++const {Toolbox} = require("devtools/client/framework/toolbox");
++
++async function setupExtensionDebuggingToolbox(extensionId, {openToolbox} = {}) {
++  DebuggerServer.init();
++  DebuggerServer.registerAllActors();
```

Comment:
> You can't just remove that part, because it's not called by default. https://searchfox.org/mozilla-central/search?q=symbol:%23initDebuggerServer&redirect=false, this is used only in two functions for Perf and WebConsole tabs. The method is defined but not used anywhere.

> If you rewrite the test as an XPCshell test, that head file is not loaded anymore.